minio/minio
# Allow `null` versions to be replicated

for pre-existing objects present in a bucket
prior to enabling existing object replication.

## Description


## Motivation and Context
Provide an easy way to sync pre-existing objects present on a bucket prior to enabling `existing-objects` replication rule

## How to test this PR?
Upload files to a bucket without versioning, then enable versioning and "existing-objects" replication. `null` versions should also get replicated

## Types of changes 

## Checklist:
